api/proto/config/netcmn.proto

```diff
@@ -98,6 +98,9 @@ enum NetworkType {
   CryptoV4 = 24;
   CryptoV6 = 26;
   CryptoEID = 14;
+  V4Only = 5;
```

Comment:
Nit, but I wouldn't mind seeing these in numeric order.
Also, I think it would make sense to put a comment at the V4 and V6 ones saying "legacy - interpreted as same as DualStack".

Reply:
numbered V4Only, V6Only & DualV4V6  with 7, 8, 9 respectively